addons/events/fnc_addBISPlayerEventHandler.sqf

```diff
@@ -0,0 +1,71 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_addBISPlayerEventHandler
+
+Description:
+    Adds an engine event handler just to the controlled entity.
+
+Parameters:
+    _eventType     - Type of event to add. Can be any engine (not mission) event. <STRING>
```

Comment:
There are more non-Mission event handlers in the game: [Scripted Event Handlers](https://community.bistudio.com/wiki/Arma_3:_Scripted_Event_Handlers) or [User Interface Event Handlers](https://community.bistudio.com/wiki/User_Interface_Event_Handlers) to name some.

This works with anything from `addEventHandler` only.

```suggestion
    _eventType     - Type of event to add. Can be any event supported by addEventHandler. <STRING>
```